app/split_integer.py

```diff
@@ -1,7 +1,13 @@
+
 def split_integer(value: int, number_of_parts: int) -> list:
     parts = []
     for parts_left in range(number_of_parts, 0, -1):
         next_number = value // parts_left
         parts.append(value // parts_left)
+        print(parts_left, parts, value)
```

Comment:
The `print` statement is likely used for debugging purposes. Consider removing it or using logging for production code.